packages/blockchain-extension/extension/commands/UserInputUtil.ts

```diff
@@ -425,7 +425,12 @@ export class UserInputUtil {
             placeHolder: prompt
         };
 
-        return await vscode.window.showQuickPick(quickPickItems, quickPickOptions);
+        if (quickPickItems.length === 0) {
+            throw new Error('No contracts found to instantiate');
```

Comment:
UserInputUtil.showChaincodeAndVersionQuickPick is also called from the upgrade command, so the error message shouldn't mention 'instantiate'. I think a generic 'No contracts found' should suffice.

Reply:
Yup, i agree